test/XRP/fakes/fake-web-socket-network-client.ts

```diff
@@ -38,6 +39,9 @@ export class FakeWebSocketNetworkClientResponses {
     public readonly getSubscribeResponse: Result<
       StatusResponse
     > = FakeWebSocketNetworkClientResponses.defaultSubscribeResponse(),
+    public readonly getUnsubscribeResponse: Result<
```

Comment:
we can probably call this just `unsubscribeResponse` and the other one `subscribeResponse`.... the only reason "get" appears in the other responses is because "get" is actually in the method name.